Tests/docker_native_image_config.json

```diff
@@ -194,16 +194,6 @@
                 "native:dev",
                 "native:candidate"
             ]
-        },
-        {
-            "id":"CrowdstrikeFalcon",
-            "reason":"urllib",
-            "ignored_native_images":[
-                "native:8.4",
-                "native:8.3",
```

Comment:
why was this skipped on all the native images in the first place? Only dev was a problem

Reply:
you are right, @tkatzir fyi